samples/sqs/native-integration/Sqs_5/Sender/Program.cs

```diff
@@ -30,6 +30,7 @@ static async Task Main()
                         //{"S3BodyKey", new MessageAttributeValue {DataType = "String", StringValue = "s3bodykey"}}, // optional for native integration
                         {"SomeRandomKey", new MessageAttributeValue {DataType = "String", StringValue = "something-random"}},
                         {"AnotherRandomKey", new MessageAttributeValue {DataType = "String", StringValue = "something-else-thats-random"}},
+                        //{"ReplyToAddress", new MessageAttributeValue {DataType = "String", StringValue = "my-native-endpoint"}}, //optional, used to demo replying back to the native endpoint, make sure to create the queue manually in AWS
```

Comment:
Is this supposed to be commented out?

Reply:
Yes since you need to manually create the queue for it to work. I considered to automate that but choose to instead just add instructions to uncomment here: https://github.com/Particular/docs.particular.net/pull/5442/files#diff-378d2f5fbb57706f605aa292f17a3ee63d6bb809f951cdd0989b2978ac259a0fR33

Is that to lazy?